lib/elixir/lib/supervisor.ex

```diff
@@ -120,9 +120,9 @@ defmodule Supervisor do
 
     * `:id` - any term used to identify the child specification internally by
       the supervisor; defaults to the given module. This key is required.
-      For Supervisors, in the case of conflicting `:id` values, the supervisor
+      For [supervisors](`Supervisor`), in the case of conflicting `:id` values, the supervisor
```

Comment:
There is no need to relink to the module itself here, let's just remove the mistaken capital letter:

```suggestion
      For supervisors, in the case of conflicting `:id` values, the supervisor
```